tests/scripts/run_performance.sh

```diff
@@ -120,6 +122,8 @@ run_device_perf_models() {
         env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/metal_BERT_large_11/tests -m $test_marker
 
         env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/falcon7b_common/tests -m $test_marker
+
+        env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/wormhole/distilbert/tests -m $test_marker
```

Comment:
Referring to the [document](https://github.com/tenstorrent/tt-metal/blob/53c32c0c0da926f97bd0eb042e70fd54c2866f44/docs/source/ttnn/ttnn/demos.rst#L31:~:text=Please%20add%20your%20test%20to%20the%20function%20run_device_perf_models()%20within%20run_performance.sh%20to%20collect%20and%20report%20on%20device%20metrics.%20Only%20tests%20that%20have%20been%20marked%20with%20the%20annotation%20%40pytest.mark.models_device_performance_bare_metal%20will%20be%20scheduled%20to%20run), the pytest fixture `@pytest.mark.models_device_performance_bare_metal `has been added only to `test_distilbert_perf_device`, so it will consider only `test_distilbert_perf_device` and ignore `test_performance_distilbert_for_qa`